### PR TITLE
feat: Scooter onboarding

### PR DESCRIPTION
**Pull request by KristianSelnas**

### CODE REVIEW

File: src/stacks-hierarchy/Root_MobileTokenOnboarding/MobileTokenOnboarding_FlexibilityInfoScreen.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import TravelPlanning from '@atb/assets/svg/color/images/TravelPlanning';
+import TravelPlanning from '@atb/assets/svg/color/images/light/TravelPlanning';
```
Comment (KristianSelnas) [inline src/stacks-hierarchy/Root_MobileTokenOnboarding/MobileTokenOnboarding_FlexibilityInfoScreen.tsx:1]: This image was moved, but light and dark images are currently similar. We need to devise a system for automatically selecting the correct image, as is currently done in the webshop. This is currently out of scope of this task.

Comment (rosvik) [reply src/stacks-hierarchy/Root_MobileTokenOnboarding/MobileTokenOnboarding_FlexibilityInfoScreen.tsx:1]: I agree, we should add better support for it. 

Is this the only asset that needs to be updated after the design system update? @mortennordseth

Comment (KristianSelnas) [reply src/stacks-hierarchy/Root_MobileTokenOnboarding/MobileTokenOnboarding_FlexibilityInfoScreen.tsx:1]: Yes, Morten claimed this was the only asset affected.

Comment (KristianSelnas) [reply src/stacks-hierarchy/Root_MobileTokenOnboarding/MobileTokenOnboarding_FlexibilityInfoScreen.tsx:1]: I've created an issue for this, btw: https://github.com/AtB-AS/kundevendt/issues/3621

**Review by rosvik — APPROVED** [review]

**Pull request merged**